src/Test/L0/Listener/Configuration/ConfigurationManagerL0.cs

```diff
@@ -179,6 +180,7 @@ public async Task CanEnsureConfigure()
                 Assert.True(s.AgentName.Equals(_expectedAgentName));
                 Assert.True(s.PoolId.Equals(_secondRunnerGroupId));
                 Assert.True(s.WorkFolder.Equals(_expectedWorkFolder));
+                Assert.True(s.Ephemeral.Equals(true));
```

Comment:
I snuck this in as another ephemeral test case, but it's not strictly related to the goals of this PR.